code-studio/test/js/progressTest.js

```diff
@@ -19,41 +19,31 @@ describe('progress', function () {
 });
 
 describe('bestResultLevelId', function () {
-  var serverProgress, clientProgress;
+  var progressData;
   before(function () {
-    serverProgress = {
+    progressData = {
       1: 0,
       2: 0,
       3: -50,
       4: 20,
       5: 100,
       6: 0,
-      7: 100,
-    };
-    clientProgress = {
-      6: 20,
-      7: 0,
+      7: 100
```

Comment:
If I understand this correctly, there's a sense in which this test stopped covering the progress merge behavior (since that responsibility has been moved to redux).  Is that merge covered somewhere else?